### ISSUE TITLE
Provide `run.sh` script to run in-vm Hyperfoil benchmarks easier

### ISSUE THREAD

**Issue by lampajr** [Hyperfoil/Hyperfoil#461]
**Is your feature request related to a problem? Please describe.**
This is more like an enhancement, rather than a problem.

Right now if a user wants to try out Hyperfoil needs to startup the `cli` (and or the controller server), then connect to the server from the `cli` and after that they can upload and run benchmarks.

This could be quite tricky overhead, especially if you just want to run a very simple benchmark in the `in-vm` mode, i.e., starting the local controller using `start-local` from the `cli`.

Moreover, running these kind of tests requires interactive communication with the CLI, which makes really hard to automate benchmarks runs in some custom scripts.

**Describe the solution you'd like**
I am proposing, similarly to what have been done for `wrk`/`wrk2` scripts, to create a new `run.sh` script that should make the aforemention process a lot easier.

Supported use cases:
- Run a benchmark by directly providing the benchmark file, no need to do that into 2 separate steps
- Monitor run termination
- Automatically generates the final report html

In conclusion the script will collapse `upload`, `run` and `report` into one single command that you can much more easily automate in any script.

**Describe alternatives you've considered**
n/a

**Additional context**
My idea was to create a new command extending the already exisintg `Run`, by providing a superset of available options:
- `--benchmark-file / -f`, benchmark to upload and run
- `--output / -o`, report html output file - note I cannot use `-d / --destination` as the `-d` is already used for the `description`

**Comment by johnaohara** [comment]
Would be good to add this to: https://github.com/Hyperfoil/jbang-catalog/blob/main/jbang-catalog.json

That would allow users to run benchmark script with

```
$ jbang run@hyperfoil benchmark.hf.yml
```

**Comment by lampajr** [comment]
> Would be good to add this to: https://github.com/Hyperfoil/jbang-catalog/blob/main/jbang-catalog.json

+1, I created this https://github.com/Hyperfoil/jbang-catalog/issues/2 to keep truck of this - happy to do it once this got merged.
